lib/tdex/registry.ts

```diff
@@ -17,6 +17,21 @@ function getRegistryURL(network: NetworkNames): string {
   return Registries[network] || Registries[NetworkNames.MAINNET]
 }
 
+async function respondsWithCorrectVersion(
+  provider: TDEXv2Provider,
+): Promise<boolean> {
+  try {
+    const url = provider.endpoint + '/v1/info'
+    const res = await axios.post(url, { list_services: '' })
```

Comment:
shall we define the header `Content-Type: application/json` too?

Reply:
No, it throws an error 🤷‍♂️

Reply:
weird! 🤷‍♂️